src/main/java/zowe/client/sdk/zosfiles/uss/input/CreateParams.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package zowe.client.sdk.zosfiles.uss.input;
+
+import java.util.Optional;
+
+/**
+ * Utility class for USS methods
```

Comment:
The title description is a bit misleading. Utility class provides a set of static helper methods that will be reused throughout the project. This class is a class that is a container of parameter values for the create api. Can you try to update the description. Look at the other input parameter classes used in other packages for examples.